src/handwritten/_installer_metadata.json

```diff
@@ -15,13 +15,14 @@
     ],
     "Linux": [
         {
-            "Version": "24.5.0",
-            "Release": "2024Q3",
+            "Version": "24.8.0",
+            "Release": "2024Q4",
             "_comment": "Location url must be of the format 'https://download.ni.com/support/softlib/MasterRepository/LinuxDrivers<Release>/NILinux<Release>DeviceDrivers.zip'. Any change to the format will require a change in the code.",
-            "Location": "https://download.ni.com/support/softlib/MasterRepository/LinuxDrivers2024Q3/NILinux2024Q3DeviceDrivers.zip",
+            "Location": "https://download.ni.com/support/softlib/MasterRepository/LinuxDrivers2024Q4/NILinux2024Q4DeviceDrivers.zip",
             "supportedOS": [
                 "ubuntu 20.04",
                 "ubuntu 22.04",
+                "ubuntu 24.04",
```

Comment:
This change didn't make it to the generated copy

Reply:
opps, forgot bout that. updated :)